rebabel_format/converters/elan.py

```diff
@@ -131,7 +131,7 @@ def process_tier(self, tier_name, tier, parent_relation):
                         self.set_feature(i, 'ELAN:'+tier_name, 'str', t)
 
 class EAFWriter(Writer):
-    identifier = 'elan'
+    identifier = 'eaf'
```

Comment:
I was actually considering normalizing this in the opposite direction, such that both identifiers and the tier name were all `elan`. (I'm thinking of similarly normalizing all readers and writers such that identifier and the tier are the same and are lowercase, so the `FlexText` tier would become `flextext`.)